cdk/lib/__snapshots__/prism.test.ts.snap

```diff
@@ -415,22 +415,22 @@ dpkg -i /prism/prism.deb",
           ],
           "Version": "2012-10-17",
         },
-        "PolicyName": "GetDistributablePolicyC6B4A871",
+        "PolicyName": "GetDistributablePolicyPrismBDF1EF88",
         "Roles": Array [
           Object {
-            "Ref": "InstanceRole",
+            "Ref": "InstanceRolePrism96D154B7",
           },
         ],
       },
       "Type": "AWS::IAM::Policy",
     },
-    "GuHttpsEgressSecurityGroupF63CDA96": Object {
+    "GuHttpsEgressSecurityGroupPrism4A68FA56": Object {
       "Properties": Object {
-        "GroupDescription": "Allow all outbound traffic on port 443",
+        "GroupDescription": "Allow all outbound HTTPS traffic",
```

Comment:
This will result in a replacement 😢 . I wonder if guardian/cdk could send a warning on this event? Though maybe the major version number change is enough of a signal?

Reply:
Hmm, good point. We have (at least) two different types of breaking change:

1. We've changed the API and you'll need to update some code, but it shouldn't cause any breaking changes to the underlying infrastructure 
2. We've changed something which will have a significant impact on the underlying infrastructure (e.g. replacing a security group, updating load balancer version)

I think everyone will be familiar with type 1, but I wonder if a major version bump will be enough to prepare them for changes of type 2... hopefully these will be rare events once we have our production-ready version though!